libse/SubtitleFormats/SubViewer20.cs

```diff
@@ -105,57 +95,73 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
-                            expecting = ExpectingLine.Text;
-                        }
-                        catch
-                        {
-                            expecting = ExpectingLine.TimeCodes;
-                        }
+                        int startHours = int.Parse(parts[0]);
+                        int startMinutes = int.Parse(parts[1]);
+                        int startSeconds = int.Parse(parts[2]);
+                        int startMilliseconds = int.Parse(parts[3]);
+                        int endHours = int.Parse(parts[4]);
+                        int endMinutes = int.Parse(parts[5]);
+                        int endSeconds = int.Parse(parts[6]);
+                        int endMilliseconds = int.Parse(parts[7]);
+                        paragraph.StartTime = new TimeCode(startHours, startMinutes, startSeconds, startMilliseconds);
+                        paragraph.EndTime = new TimeCode(endHours, endMinutes, endSeconds, endMilliseconds);
+                        expecting = ExpectingLine.Text;
                     }
-                }
-                else
-                {
-                    if (expecting == ExpectingLine.Text)
+                    catch
                     {
-                        if (line.Length > 0)
-                        {
-                            string text = line.Replace("[br]", Environment.NewLine);
-                            text = text.Replace("{\\i1}", "<i>");
-                            text = text.Replace("{\\i0}", "</i>");
-                            text = text.Replace("{\\i}", "</i>");
-                            text = text.Replace("{\\b1}", "<b>'");
-                            text = text.Replace("{\\b0}", "</b>");
-                            text = text.Replace("{\\b}", "</b>");
-                            text = text.Replace("{\\u1}", "<u>");
-                            text = text.Replace("{\\u0}", "</u>");
-                            text = text.Replace("{\\u}", "</u>");
-
-                            paragraph.Text = text;
-                            subtitle.Paragraphs.Add(paragraph);
-                            paragraph = new Paragraph();
-                            expecting = ExpectingLine.TimeCodes;
-                        }
+                        _errorCount++;
                     }
                 }
+                else if (expecting == ExpectingLine.Text && lineTrimmed.Length > 0)
+                {
+                    paragraph.Text = DecodeFormatting(lineTrimmed);
+                    subtitle.Paragraphs.Add(paragraph);
+                    paragraph = new Paragraph();
+                    expecting = ExpectingLine.TimeCodes;
+                }
+                else if (lineTrimmed.Length > 0) // Do not count empty lines
+                {
+                    _errorCount++;
+                }
             }
+            // Do not include metadata

```

Comment:
Hm, error handling should be improved... try to see what happens on non subviewer 2 file